_posts/2015-08-18-Test-Tab-Now-Available-in-Apiary.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Integrating API Design with Continuous Integration
+title: Continuously Integrate Your API Design
 author: davita
-excerpt: "Apiary now the only API design platform with tools for every stage of the API lifecycle"
+excerpt: "Apiary is now the only API design platform with tools for every stage of the API lifecycle"
```

Comment:
@kuba-kubula 
Stupid, but should this have punctuation at the end of the sentence?